deploy/config

```diff
@@ -58,15 +58,15 @@ sunbird_azure_storage_key:                      # Azure account storage key
 sunbird_azure_storage_account:                  # Azure account storage name
 
 #Sunbird System Initialisation - First organisation and user in Sunbird will be created with these details
-sunbird_custodian_tenant_name: 					# Custodian Organisation Name
-sunbird_custodian_tenant_description: 			# Custodian Organisation Description
-sunbird_custodian_tenant_channel: 				# Custodian Organisation Channel
-sunbird_root_user_firstname: 					# first name of the Sunbird admin user
-sunbird_root_user_lastname: 					# last name of the Sunbird admin user(optional field)
-sunbird_root_user_username: 					# username to be used for login to Sunbird
-sunbird_root_user_password: 					# password to be used for login to Sunbird
-sunbird_root_user_email: 						# email address of the Sunbird admin user
-sunbird_root_user_phone: 						# mobile number of the Sunbird admin user
+sunbird_init_custodian_tenant_name: 				# Custodian Organisation Name
```

Comment:
why is the prefix `sunbird_init_`

Reply:
@kochhar - `init` was added to the variable names to indicate the stage of installation where these variables are used. We got this review comment during last monday showcase.
At present, the variable names in config file are
```
sunbird_init_custodian_tenant_name
sunbird_init_custodian_tenant_description
sunbird_init_custodian_tenant_channel
sunbird_init_admin_user_firstname
sunbird_init_admin_user_lastname
sunbird_init_admin_user_username
sunbird_init_admin_user_password
sunbird_init_admin_user_email
sunbird_init_admin_user_phone
```

Please suggest if there are any changes needed.